src/cpp/valeurs_usage/include/antares-xpansion/valeurs_usage/ValeursUsage.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "antares-xpansion/benders/benders_core/BendersMathLogger.h"
+#include "antares-xpansion/benders/benders_core/SubproblemCut.h"
+#include "antares-xpansion/benders/benders_core/SubproblemWorker.h"
+#include "antares-xpansion/benders/output/JsonWriter.h"
+#include "antares-xpansion/xpansion_interfaces/ILogger.h"
+
+using ConstraintMap = std::map<std::string /*constraint name*/, std::vector<double> /*rhs values*/>;
+using AreaConstraintMaps = std::map<std::string /*area name*/, ConstraintMap>;
+using SubPbConstraintMaps = std::map<std::string /*subPb name*/, AreaConstraintMaps>;
+
+struct ScenarioAndWeek
+{
+    int scenario;
+    int week;
+
+    bool operator<(const ScenarioAndWeek& other) const
+    {
+        return std::tie(scenario, week) < std::tie(other.scenario, other.week);
+    }
+};
+
+/// @brief Class to compute the valeurs d'usage
+class ValeursUsage
```

Comment:
In practice here we are not computing the `ValeursUsage` but what we call `TransitionGains` which are then used in post processing to compute with another algorithm the usage values